.github/workflows/release.yaml

```diff
@@ -0,0 +1,19 @@
+name: Release CSI-Powerstore
+# Invocable as a reusable workflow
+# Can be manually triggered
+on:  # yamllint disable-line rule:truthy
+  workflow_call:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to release (major, minor, patch), Example: 1.x.x'
+        default: '1.1.1'
```

Comment:
Is there any chance that the default of 1.1.1 will get wrongly used?

Reply:
I have removed the defaults which I set for testing